core/src/test/scala/unit/kafka/log/LogTest.scala

```diff
@@ -4447,9 +4504,10 @@ class LogTest {
 
   private def recoverAndCheck(config: LogConfig,
                               expectedKeys: Iterable[Long],
-                              expectDeletedFiles: Boolean = true): Log = {
+                              expectDeletedFiles: Boolean = true,
```

Comment:
It seems that all callers set expectDeletedFiles to false. So, do we need this param?

Reply:
I think you meant the `hadCleanShutdown` parameter I added. I did not add the `expectDeletedFiles` parameter.
I will remove the new parameter and add a comment here to indicate that the method always assumes we had a hard reset.

Reply:
Done